tests/testSeparableXYTransform.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+#
+# LSST Data Management System
+# Copyright 2015 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""
+Tests for lsst.afw.geom.SeparableXYTransform and Functor classes.
+"""
+import unittest
+import numpy as np
+import numpy.random as random
+import lsst.utils.tests
+import lsst.afw.geom as afwGeom
+
+def num_deriv(func, x, eps=1e-7):
+    h = eps*max(abs(x), 1)
+    xp = x + h
+    dx = xp - x
+    return (func(x + dx) - func(x))/dx
+
+class FunctorTestCase(unittest.TestCase):
+    def setUp(self):
+        self.pars = [('', 5, 30), ('', 2, 30, 4000)]
+        self.funcs = [afwGeom.LinearFunctor,
+                      afwGeom.EdgeRolloffFunctor]
+        self.xvals = np.logspace(-8, 1, 10)
+    def tearDown(self):
+        pass
```

Comment:
There's no need to explicitly tear things down as they'll be deleted when the test goes out of scope.  The exception is things for which we're tracking memory, as testing for leaks occurs earlier.